common/lib/xmodule/xmodule/templates/problem/formularesponse.yaml

```diff
@@ -5,40 +5,52 @@ metadata:
 data: |
   <problem>
   <p>
-  A math expression input problem accepts a line of text representing a mathematical expression from the
-  student, and evaluates the input for equivalence to a mathematical expression provided by the 
-  grader. Correctness is based on numerical sampling of the symbolic expressions.
+  In math expression input problems, learners enter text that represents a
+  mathematical expression into a field, and text is converted to a symbolic
+  expression that appears below that field. Learners can see 
+  <a href="http://edx.readthedocs.org/projects/edx-guide-for-students/en/latest/SFD_mathformatting.html" target="_blank">
+  Entering Mathematical and Scientific Expressions</a> in the <i>edX Guide for
+  Students</i> for information about how to enter text into the field.
   </p>
   <p>
-  The answer is correct if both the student provided response and the grader's mathematical
-  expression are equivalent to specified numerical tolerance, over a specified range of values for each
-  variable.
+  Math expression problems can include unknown variables and relatively
+  complicated symbolic expressions. The grader uses a numerical sampling to
+  determine whether the student’s response matches your math expression, to a
+  specified numerical tolerance. You must specify the allowed variables in the
+  expression as well as the range of values for each variable.
   </p>
+  <p>
+  To create these problems, you use MathJax to change your plain text into
+  "beautiful math." For more information about how to use MathJax in Studio,
+  see <a href="http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/exercises_tools/mathjax.html" target="_blank">
+  A Brief Introduction to MathJax in Studio</a> in <i>Building and Running an edx
+  Course</i>.  
+  </p>
+  <p>You can use the following example problems as models.</p>
+  <p>_____________________________________________________________________________</p>
```

Comment:
The only problem with that is that ``<hr>`` will only provide a visible separator before the user opens the problem for editing, and I think it would be useful to preserve the separator while the user is working with the problem.